apel/common/message_schemas.py

```diff
@@ -0,0 +1,149 @@
+                    "LocalUser",
+                    "LocalGroup",
+                    "GlobalUserName",
+                    "FQAN",
+                    "IPVersion",
+                    "IPCount"
+                ],
+                "properties": {
+                    "MeasurementTime": {
+                        "$id": "#/properties/UsageRecords/items/properties/MeasurementTime",
+                        "type": "integer",
+                        "title": "The MeasurementTime Schema",
+                        "examples": [
+                            1536070129
+                        ]
+                    },
+                    "SiteName": {
+                        "$id": "#/properties/UsageRecords/items/properties/SiteName",
+                        "type": "string",
+                        "title": "The SiteName Schema",
+                        "examples": [
+                            "TEST-SITE"
+                        ],
+
+                    },
+                    "CloudComputeService": {
+                        "$id": "#/properties/UsageRecords/items/properties/CloudComputeService",
+                        "type": [
+                            "string",
+                            "null"
+                        ],
+                        "title": "The CloudComputeService Schema",
+                        "examples": [
+                            "TEST-CLOUD"
+                        ],
+
+                    },
+                    "CloudType": {
+                        "$id": "#/properties/UsageRecords/items/properties/CloudType",
+                        "type": "string",
+                        "title": "The CloudType Schema",
+                        "examples": [
+                            "caso/1.1.0  (OpenStack)",
+                            "OpenNebula oneacct-export/0.4.6"
+                        ],
+
+                    },
+                    "LocalUser": {
+                        "$id": "#/properties/UsageRecords/items/properties/LocalUser",
+                        "type": "string",
+                        "title": "The LocalUser Schema",
+                        "examples": [
+                            "User1"
+                        ],
+
+                    },
+                    "LocalGroup": {
+                        "$id": "#/properties/UsageRecords/items/properties/LocalGroup",
+                        "type": "string",
+                        "title": "The LocalGroup Schema",
+                        "examples": [
+                            "Group1"
+                        ],
+
+                    },
+                    "GlobalUserName": {
+                        "$id": "#/properties/UsageRecords/items/properties/GlobalUserName",
+                        "type": "string",
+                        "title": "The GlobalUserName Schema",
+                        "examples": [
+                            "GlobalUser1"
+                        ],
+
+                    },
```

Comment:
GlobalUserName should be nullable because the tenancy model in OpenStack is based on projects and there will be IPs allocated to a project, not owned by any user.